app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -935,9 +951,14 @@ class HomeActivityTest {
         )
       ).perform(click())
       intended(hasComponent(TopicActivity::class.java.name))
-      intended(hasExtra(TopicActivity.getProfileIdKey(), internalProfileId1))
       intended(hasExtra(TopicActivity.getTopicIdKey(), FRACTIONS_TOPIC_ID))
       intended(hasExtra(TopicActivity.getStoryIdKey(), FRACTIONS_STORY_ID_0))
+      it.onActivity { it1 ->
+        assertThat(
+          it1.intent.extractCurrentUserProfileId()
+            .internalId
+        ).isEqualTo(internalProfileId1)
```

Comment:
```suggestion
        assertThat(activity.intent.extractCurrentUserProfileId().internalId)
          .isEqualTo(internalProfileId1)
```

This can wrap more cleanly. Ditto everywhere else in the PR.

Reply:
Updated.